tools/serverpod_cli/lib/src/analyzer/entities/entity_parser/entity_parser.dart

```diff
@@ -169,19 +165,65 @@ class EntityParser {
     );
     var scope = _parseClassFieldScope(value);
     var parentTable = _parseParentTable(value);
+    var scalarField = _parseScalarField(value, fieldName);
     var isEnum = _parseIsEnumField(value);
 
-    return SerializableEntityFieldDefinition(
-      name: fieldName,
-      scope: scope,
-      type: typeResult.type..isEnum = isEnum,
-      parentTable: parentTable,
-      documentation: fieldDocumentation,
-    );
+    return [
+      if (_isRelation(value))
+        SerializableEntityFieldDefinition(
+          name: _createScalarFieldName(fieldName),
+          scope: SerializableEntityFieldScope.all,
```

Comment:
Question: Should this be configurable so that it is only available on the server? What if the user only wants the id and relation serverside.

Reply:
Per discussion we postpone this, until we resolve the `scope` syntax, see https://github.com/serverpod/serverpod/issues/1142.